extension/src/setup/index.ts

```diff
@@ -325,6 +325,30 @@ export class Setup
     return this.updateStudioAndSend()
   }
 
+  public removeStudioUrl() {
+    if (!this.getCliCompatible()) {
+      return
+    }
+
+    return this.studio.removeStudioUrl(this.dvcRoots)
+  }
+
+  public async saveStudioUrl() {
+    const cwd = this.getCwd()
```

Comment:
Did look into using `getCwd()` for `saveStudioAccessToken` and it does look like we can use it without issues. Will update (plus maybe remove the dvc compatible check as well) in a followup!

Reply:
Discussed synchronously, but we'll keep the dvc compatible check for failsafe reasons.